apps/antidote_crdt/src/antidote_crdt_map_rr.erl

```diff
@@ -223,8 +223,7 @@ is_operation(Operation) ->
         {update, {{_Key, Type}, Op}} ->
             antidote_crdt:is_type(Type) andalso antidote_crdt:is_operation(Type, Op);
         {update, Ops} when is_list(Ops) ->
-            distinct([Key || {Key, _} <- Ops]) andalso
-                lists:all(fun(Op) -> is_operation({update, Op}) end, Ops);
+            lists:all(fun(Op) -> is_operation({update, Op}) end, Ops);
```

Comment:
There are a few more calls for `distinct` check further down, I think they can (and should) also be removed.